src/bitmessagekivy/tests/test_create_random_address.py

```diff
@@ -63,13 +66,11 @@ def test_set_default_address(self):
         # This is for opening side navbar
         self.open_side_navbar()
         # Click to open Address Dropdown
-        self.cli.wait_click('//NavigationItem[0]/CustomSpinner[0]', timeout=5)
+        self.cli.wait_click('//IdentitySpinner[0]', timeout=1)
```

Comment:
Here also, i added `name:` in  `IdentitySpinner` but getting assertion error and can not get `name` and `id`